### ISSUE TITLE
Update tag 1.3.0 on About page

### ISSUE THREAD

**Issue by vildead** [elixir-luxembourg/daisy#144]
The About page is still showing version 1.2.0 despite tag 1.3.0 (develop - commit d917db9) being deployed. 
 See daisy-test VM

**Comment by evescat** [comment]
also about ABOUT page: please updated the user guide hyperlink in ABOUT tab, currently it directs to how-to-cards manual, which does not exit.

**Comment by jLebioda** [comment]
@evescat thanks, I've already addressed that - the fix is already merged, and will be included in the next release

@vildead it looks like the dynamic code to detect the version (web/views/context_processors.py) found previous package; maybe it's just a matter of calling once again `/usr/local/bin/pip install -e /home/daisy/daisy --upgrade`

**Comment by vildead** [comment]
@jLebioda  my apologies! Deployment of daisy-test is failing because of **git pull** :facepalm: -> not upgrading. I will fix it today and close this issue:)

**Comment by jLebioda** [comment]
no worries @vildead :)

**Comment by vildead** [comment]
I am sorry @jLebioda , it looks like something more. 

https://github.com/elixir-luxembourg/daisy/blob/d917db9a1799d61766200e2de7a50244e7675f13/setup.py#L49

Running `pip upgrade` gives me
```
Found existing installation: elixir-daisy 1.2.0
    Can't uninstall 'elixir-daisy'. No files were found to uninstall.
  Running setup.py develop for elixir-daisy
```
I guess the release 1.3.0 is has some issues :/

**Comment by jLebioda** [comment]
I think I know what might be wrong  - it looks like that in the last release we actually forgot to bump the version 🤦 and it's still 1.2.0 in the setup.py (so the main file which describes the DAISY as a package), even though it's already 1.3.0

**Comment by vildead** [comment]
@jLebioda , Indeed. the features described in 1.3.0 release are there. 
Just curious - should npm version match release version of the whole app? Or is it independent?
https://github.com/elixir-luxembourg/daisy/blob/382cf9c5301d704659eb90078e164bd80bc76050/web/static/vendor/package-lock.json#L3

**Comment by jLebioda** [comment]
@vildead  It's completely independent and the version from that file does not matter at all at this point (it would if we released front-end package). It's the cruel conventions that force us to put the version there :D

BTW, `private: true` might be added to the `package-lock.json` file, too

**Comment by vildead** [comment]
@jLebioda can I help in this matter anyhow? This is obviously caused by missing bumb-version guideline/doc rather than one time error.

**Comment by jLebioda** [comment]
@vildead I think we don't have guidelines for releasing the code (only for new deployments and upgrades). To be honest, I don't know what's the _best_ way to act now - I think we could just roll out 1.3.1 from current develop;
@neoflex @pinarpink do you agree with that?

**Comment by vildead** [comment]
fixed on release [1.4.0](https://github.com/elixir-luxembourg/daisy/tree/1.4.0)

Closing
